source/GameAction.cpp

```diff
@@ -40,10 +40,11 @@ namespace {
 	{
 		// Maps are not transferrable; they represent the player's spatial awareness.
 		int mapSize = outfit->Get("map");
+		bool showMinable = outfit->Get("show minable");
```

Comment:
Probably should call it `"map minables"`.

Reply:
Yea that's better. I'll push an update after dinner.